SRanipalExtTrackingModule/SRanipalTrackingInterface.cs

```diff
@@ -0,0 +1,416 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Threading;
+using ViveSR;
+using ViveSR.anipal;
+using ViveSR.anipal.Eye;
+using ViveSR.anipal.Lip;
+using VRCFaceTracking;
+using VRCFaceTracking.Assets.UI;
+using VRCFaceTracking.Params;
+
+namespace SRanipalExtTrackingInterface
+{
+    public class SRanipalExtTrackingInterface : ExtTrackingModule
+    {
+        LipData_v2 lipData = default;
+        EyeData_v2 eyeData = default;
+        private static bool eyeEnabled = false, lipEnabled = false;
+
+        private static CancellationTokenSource _cancellationToken;
+        
+        public override (bool SupportsEye, bool SupportsExpressions) Supported => (true, true);
+
+        public override (bool eyeSuccess, bool expressionSuccess) Initialize(bool eyeAvailable, bool expressionAvailable)
+        {
+            // Look for SRanipal assemblies here. Placeholder for unmanaged assemblies not being embedded in the dll.
+            Directory.SetCurrentDirectory(Utils.CustomLibsDirectory + "\\ModuleLibs");
```

Comment:
This looks in a subfolder for the SRanipal libs, though if we plan on embedding it into the SRanipal module maybe this won't be needed?